manager/integration/tests/test_node.py

```diff
@@ -983,6 +983,29 @@ def test_node_controller_sync_disk_state(client):  # NOQA
                                      CONDITION_STATUS_TRUE)
 
 
+@pytest.mark.skip(reason="TODO")
+@pytest.mark.node  # NOQA
+@pytest.mark.mountdisk # NOQA
+def test_node_default_disk_addedback_with_extra_disk_unmounted(client):  # NOQA
+    """
+    [Node] Test adding default disk back with extra disk is unmounted
+    on the node
+
+    1. Attached a new (Linux) disk to node 1.
+    2. Create an schedulable extra Longhorn disk on node 1 with
+    this extra new disk.
+    3. Delete the default disk on node 1.
+    4. Unmount the extra disk on node 1.
+    And wait for it becoming "Unschedulable".
+    5. Create and add the default disk back on node 1.
+    6. Wait and verify the default disk should become "Schedulable".
+    7. Mount extra disk back on node 1.
+    8. Wait and verify this extra disk should become "Schedulable".
+    9. Clean the extra disk on node 1.
```

Comment:
Does "clean" mean deleting the disk?

Reply:
Yes, sorry about the confusion. I will update it. Thanks.